src/identity/components/EthBalance.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { EthBalance } from './EthBalance';
+import { WagmiProvider, createConfig, http } from 'wagmi';
+import { baseSepolia } from 'viem/chains';
+
+const wagmiConfig = createConfig({
+  chains: [baseSepolia],
+  transports: {
+    [baseSepolia.id]: http(),
+  },
+});
+
+const meta = {
+  title: 'Identity/EthBalance',
+  component: EthBalance,
+  decorators: [
+    (Story) => {
+      return (
+        <WagmiProvider config={wagmiConfig}>
+          <QueryClientProvider client={new QueryClient()}>
+            <Story />
+          </QueryClientProvider>
+        </WagmiProvider>
```

Comment:
Should we move this to preview.ts as well?

Reply:
I'd favor explicitness because:

- Not every components needs to be wrapped inside
- Some components only need to be wrapped inside `QueryClientProvider`

More verbose but it's clear what's needed when reading stories.